### ISSUE TITLE
Example isn't following IWC repository (yet?)

### ISSUE THREAD

**Issue by mvdbeek** [AustralianBioCommons/iwc-template#1]
Thanks for getting this started @supernord!

There are no workflows at the root of the repository, the structure is outlined here: https://github.com/galaxyproject/iwc/blob/main/workflows/README.md#structure-of-the-directory

**Comment by supernord** [comment]
Thanks @mvdbeek 😄 

I made the following changes based on the IWC repository: https://github.com/AustralianBioCommons/iwc-template/commit/5db7acc4c72779d1091bd19522db266fc0f740e6

Based on your comment I've also realised there are two templates that could be created:
- a single workflow repository that would match the expectations of the IWC (e.g. https://github.com/iwc-workflows/pox-virus-amplicon), and also
- a template to help create a new workflow collection based on the IWC approach

maybe both approaches are useful?

**Comment by mvdbeek** [comment]
Yes, I think so. I would probably not expect the single repository per workflow style to catch on: We do automatically create these on merging something to the main branch of https://github.com/galaxyproject/iwc, and that allows users to have anywhere from one to many workflows in a single place, making updates and other maintenance tasks easier, while still being able to make proper releases on github and to the various workflow registries and Galaxy instances. I would say this is also a more flexible setup if you want to e.g. deposit workflows with additional data and data processing for your manuscript, as all you need to do is put your workflow(s) in the workflows directory.
